src/doc/rustc/src/targets/index.md

```diff
@@ -0,0 +1,14 @@
+# Targets
+
+`rustc` is a cross-compiler by default. This means that you can use any compiler to build for any
+architecture. The list of *targets* are the possible architectures that you can build for.
+
+You can see the API docs for a given target
+[here](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_back/target/struct.Target.html), all
```

Comment:
This line seems a little off. My reading implies "this is the standard library available under any given target", but it's more like "these are the things that a target can specify".